src/openrct2/world/Banner.cpp

```diff
@@ -309,3 +335,12 @@ void BannerElement::ResetAllowedEdges()
 {
     flags |= 0b00001111;
 }
+
+Banner* get_banner(BannerIndex id)
```

Comment:
Shouldn't we call this GetBanner instead? I know that a lot of code still uses underscores but that is not what we want if I'm no mistaken.